code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -1712,7 +1712,7 @@
 		return 1
 
 	M.apply_radiation(5, RAD_INTERNAL)
-	if(prob(20))
+	if(prob(20) && !istype(M, /mob/living/carbon/slime))
```

Comment:
While this definitely works, you could consider overriding `advanced_mutate()` for slimes instead

Reply:
I personally think its best here since it could be used to deal with other troublesome mob types in the future. So rather than having it in different places you keep it in one central location.

Reply:
Nope, that's a bad OOP practice and it makes the code a mess.

Reply:
Using istypes means that you have to search all over the code for the thing that's making slimes have the behavior, as opposed to just looking in the individual mob's codefile for the proc override.

Do you have any familiarity with the concept of object oriented programming?

Reply:
Yeah, it's something that needs changing and I just took the easy route at the time I did it. I can fix it when I get a chance.